src/views/Requests.svelte

```diff
@@ -51,28 +66,38 @@ function selectProvider(userId) {
     provider: userId,
     size: null,
   })
+
+  filteredRequestsByProviding()
```

Comment:
As with the `selectCreator()`, the `selectProvider()` isn't limited to selecting requests where _I'm_ the provider, so if the `filteredRequestsByProviding()` GA tracking event is intended to track uses of the "My Commitments" button, then a new `showMyCommitments()` method would probably be worthwhile.

Given both of those changes, the `RequestQuickFilter` would end up looking something like this:
```
<RequestQuickFilter {requestFilter} buttonCssClass="my-1 mx-0"
                    on:all={showAll} on:my-requests={showMyRequests} on:my-commitments={showMyCommitments} />
```

Reply:
yeah, I see that too, I think this is the same as my response to `selectCreator` comment.